src/commands/manifest.ts

```diff
@@ -61,7 +61,7 @@ export default class Manifest extends Command {
         this.cloneRepo(repoUrl, fullPath, maxSatisfying)
 
         this.executeCommand('yarn', {cwd: fullPath})
-        this.executeCommand('yarn build', {cwd: fullPath})
+        this.executeCommand('yarn tsc', {cwd: fullPath})
```

Comment:
Can you explain why this didn't work with `yarn build` ?  Is it related to the use of --incremental, or something else?

https://github.com/salesforcecli/plugin-custom-metadata/blob/0434b94bf4c2a3a1a0bd4728b4bc6d99e2b28ecb/package.json#L155